qiskit_braket_plugin/providers/braket_backend.py

```diff
@@ -59,8 +83,37 @@ def acquire_channel(self, qubit: int):
     def control_channel(self, qubits: Iterable[int]):
         pass
 
-    def run(self, run_input, **options):
-        pass
+    def run(self, qobj: QasmQobj, **options):
+        # If we get here, then we can continue with running, else ValueError!
+        circuits: List[Circuit] = list(convert_qasm_qobj(qobj))
+        shots = qobj.config.shots
+
+        tasks = []
+        try:
+
+            for circuit in circuits:
+                task = self._aws_device.run(
+                    task_specification=circuit,
+                    shots=shots
+                )
+                tasks.append(task)
+
+        except Exception as ex:
+            logger.error(f'During creation of tasks an error occurred: {ex}')
+            logger.error(f'Cancelling all tasks {len(tasks)}!')
+            for task in tasks:
+                logger.error(f'Attempt to cancel {task.id}...')
+                task.cancel()
+                logger.error(f'State of {task.id}: {task.state()}.')
+            raise ex
+
+        job = AWSBraketJob(
+            job_id=qobj.qobj_id,
```

Comment:
I think `job_id` parameter here is id of running task in the backend for our referece, not`qobj_id`